mobile/src/packages/journal/libs/enums/enums.ts

```diff
@@ -0,0 +1 @@
+export { JournalApiPath } from 'shared/build/index.js';
```

Comment:
.js extension won't work on mobile
```suggestion
export { JournalApiPath } from 'shared/build/index';
```

Reply:
But we have shared/build/index.JS in many files in the mobile folder (screenshot 1), and without .js, I get an error (screenshot 2)
![image](https://github.com/BinaryStudioAcademy/bsa-2023-calmpal/assets/108797614/9c516b62-766e-4850-8ac1-7d2c876e6381)
![image](https://github.com/BinaryStudioAcademy/bsa-2023-calmpal/assets/108797614/2c9d096f-5237-4ae6-87bc-00daaf5e24a4)

Reply:
ok, I will need to check this, let's leave for now